.github/workflows/orion.yaml

```diff
@@ -9,14 +9,83 @@ env:
   CI_ROOT: ${{ secrets.ORION_CI_ROOT }}  # Env. secret variable
 
 jobs:
-  validate-user:
+  authenticate:
     runs-on: [self-hosted, orion-ready]
-    timeout-minutes: 100
+    timeout-minutes: 10
 
     steps:
     - name: Checkout ci-workflows
       uses: actions/checkout@v3
+      with:
+        repository: TerrenceMcGuinness-NOAA/ci-workflows
```

Comment:
```suggestion
      with:
        repository: NOAA-EMC/ci-workflows
```

Reply:
Yes, we need to change this once it was working.  BTW the runner itself has to be launched with the TOKEN from the global-workflow repo as do the SECRETES as they inherited to CI-workflow.